pacrunner_test.go

```diff
@@ -313,6 +313,7 @@ func TestWeekdayRange(t *testing.T) {
 	for _, test := range tests {
 		for i, weekday := range weekdays {
 			t.Run(test.name+" "+weekday.name, func(t *testing.T) {
+				t.Skip("Pending test to implement Github Releases.")
```

Comment:
What was the reason for skipping these tests?

Reply:
```
            	Error:      	Not equal: 
            	            	expected: false
            	            	actual  : true
            	Test:       	TestWeekdayRange/M-F_UTC_MON
    --- FAIL: TestWeekdayRange/M-F_UTC_SAT (0.00s)
        pacrunner_test.go:327: 
            	Error Trace:	pacrunner_test.go:327
            	Error:      	Not equal: 
            	            	expected: true
            	            	actual  : false
            	Test:       	TestWeekdayRange/M-F_UTC_SAT
    --- FAIL: TestWeekdayRange/SAT_UTC_SUN (0.00s)
        pacrunner_test.go:327: 
            	Error Trace:	pacrunner_test.go:327
            	Error:      	Not equal: 
            	            	expected: true
            	            	actual  : false
            	Test:       	TestWeekdayRange/SAT_UTC_SUN
    --- FAIL: TestWeekdayRange/SAT_UTC_SAT (0.00s)
        pacrunner_test.go:327: 
            	Error Trace:	pacrunner_test.go:327
            	Error:      	Not equal: 
            	            	expected: false
            	            	actual  : true
            	Test:       	TestWeekdayRange/SAT_UTC_SAT
FAIL
FAIL	github.com/samuong/alpaca	0.271s
ok  	github.com/samuong/alpaca/cancelable	0.003s
```

Will have to investigate more but this is what I'm seeing.

Reply:
That's really weird. I've created #46 to track this, but the problem isn't with your change, so let's disable it in this pull request for now.

I suspect this has something to do with timezones not being handled properly - I'm pretty sure it worked when I ran it in Melbourne, but now it fails on my local machine in New York.

Reply:
Added ref to issue in pending.